pkg/logql/ast.go

```diff
@@ -334,6 +334,7 @@ func (e *LineFilterExpr) String() string {
 	return sb.String()
 }
 
+/*
```

Comment:
Why is this part commented out?

Reply:
That's to run the benchmarks against the original version. I'll remove it once we are merging this patch.

Reply:
I think we can uncomment this one since we're about to merge it.

Reply:
Good catch!